lenskit-cli/src/main/java/org/lenskit/cli/util/Logging.java

```diff
@@ -91,10 +97,20 @@ public static void configureLogging(Namespace options) {
         console.setEncoder(consolePat);
         root.addAppender(console);
 
+        // figure out the root appender's level
+        Level rootLevel = logLevel;
+        if (logFileLevel.toInt() < logLevel.toInt()) {
+            rootLevel = logFileLevel;
```

Comment:
It feels to me like it might be worth saving this if for when we know if there is a logFile. There is no reason to step down for a lower log file level and then filter if the log file is disabled right?